lib/driver.js

```diff
@@ -434,26 +438,28 @@ class XCUITestDriver extends BaseDriver {
         await setLocaleAndPreferences(sim, this.opts, this.isSafari());
       });
 
-      if (this.opts.customSSLCert) {
+      if (this.opts.customSSLCert && !(await doesSupportKeychainApi(this.opts.device))) {
         const certHead = _.truncate(this.opts.customSSLCert, {length: 20});
         log.info(`Installing the custom SSL certificate '${certHead}'`);
-        try {
-          await this.opts.device.simctl.addRootCertificate(this.opts.customSSLCert, {raw: true});
-        } catch (ign) {
-          if (await hasSSLCert(this.opts.customSSLCert, this.opts.udid)) {
-            log.info(`SSL certificate '${certHead}' already installed`);
-          } else {
-            log.info(`Making sure Simulator is shut down, ' +
-              'so that SSL certificate installation takes effect`);
-            await shutdownSimulator(this.opts.device);
-            await installSSLCert(this.opts.customSSLCert, this.opts.udid);
-          }
+        if (await hasCertificateLegacy(this.opts.device, this.opts.customSSLCert)) {
+          log.info(`SSL certificate '${certHead}' already installed`);
+        } else {
+          log.info(`Making sure Simulator is shut down, ' +
+            'so that SSL certificate installation takes effect`);
+          await shutdownSimulator(this.opts.device);
```

Comment:
Does it make section to address this shutdown for older Xcode case in `appium:customSSLCert` section in the readme...?

Reply:
keychain support has been added since Xcode 11.4
I don't think many customers are still using it though %)

Reply:
oh, my comment had typo...

> Does it make section to address this shutdown for older Xcode case 

-> does it make sense to address ...

:P 

anyway, this is no matter :)